### PR TITLE
Add multiline box remove button hoc

### PR DESCRIPTION
**Pull request by pochretien**
Add example for multiple use case
add the new multilinebox remove button hoc

image

### CODE REVIEW

File: src/components/multilineBox/examples/MultilineBoxExamples.tsx [hunk 1]
```diff
@@ -23,6 +24,43 @@ const MultilineBoxWithContainer = _.compose(
...
+const MultilineBoxWithContainerAndTwoRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+    multilineBoxContainer({
+        container: {
+            tag: 'div',
+            props: {
+                className: 'mod-border p1',
+            },
+        },
+    }),
```
Comment (GermainBergeron) [inline src/components/multilineBox/examples/MultilineBoxExamples.tsx]: I'm looking at this and wouldn't it be easier to directly set the element like this:

```typescript
const MultilineBoxWithContainerAndTwoRemoveButton = _.compose(
    multilineBoxWithRemoveButton(),
    multilineBoxContainer((props: theCorrectType) => (<div className='mod-border p1'>{props.children}</div>),
...
```

File: src/components/multilineBox/examples/MultilineBoxExamples.tsx [hunk 2]
```diff
@@ -23,6 +24,43 @@ const MultilineBoxWithContainer = _.compose(
...
+    multilineBoxWithRemoveButton(),
```
Comment (GermainBergeron) [inline src/components/multilineBox/examples/MultilineBoxExamples.tsx]: You could do the same thing for the remove button (but with a default value that render your actual svg)

File: src/components/multilineBox/examples/MultilineBoxExamples.tsx (hunk 1 above)
Comment (pochretien) [reply src/components/multilineBox/examples/MultilineBoxExamples.tsx]: Updated with a global variable for each example

**Comment by pochretien** [comment]
Examples Updated
image

**Pull request merged**